core/src/main/java/org/apache/gravitino/meta/RoleEntity.java

```diff
@@ -91,16 +95,7 @@ public Map<String, String> properties() {
    */
   @Override
   public List<SecurableObject> securableObjects() {
-    // The securable object is a special kind of entities. Some entity types aren't the securable
-    // object, such as
-    // User, Role, etc.
-    // The securable object identifier must be unique.
-    // Gravitino assumes that the identifiers of the entities may be the same if they have different
-    // types.
-    // So one type of them can't be the securable object at least if there are the two same
-    // identifier
-    // entities .
```

Comment:
We don't need these comments any more?

Reply:
This is legacy comments. The interface has more accurate comments.